CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-ssm-mongodb ChangeLog
 
+## 6.0.0 - 2021-xx-xx
+
+### Added
+- Add `getKeyDescription` API.
+- Add `getKeyCount` API.
+
+### Changed
+- **BREAKING**: Store `@context` with keys.
+- **BREAKING**: Key records are indexed by both keystore ID and local ID
```

Comment:
I would like to see mention of `record.id` being dropped from the data model, this would be helpful in understanding the extent of the breaking change.  And yay!  Fare thee well `database.hash`.